crates/iceberg/src/io.rs

```diff
@@ -268,6 +268,16 @@ impl OutputFile {
             .await?)
     }
 
+    /// Delete file.
+    pub async fn delete(&self) -> Result<()> {
+        // #TODO
+        // Do we need to check if file exists?
```

Comment:
It depends on whether we want to consider it an error if the file doesn't exist.